### PR TITLE
Dev/create monthly reports controller

### PR DESCRIPTION
**Pull request by Nobuhiko-Kudo**
monthly_reportsのshowを作りました

### CODE REVIEW

File: app/controllers/monthly_reports_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class MonthlyReportsController < ApplicationController
```
Comment (freezer530000) [inline app/controllers/monthly_reports_controller.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment [:ref](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md)

File: app/helpers/monthly_reports_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module MonthlyReportsHelper
```
Comment (freezer530000) [inline app/helpers/monthly_reports_helper.rb:1]: **[rails_best_practices]**
- remove empty helpers [:ref](http://rails-bestpractices.com/posts/2011/04/09/remove-empty-helpers/)

File: config/routes.rb [hunk 3]
```diff
@@ -5,6 +5,8 @@
   get 'sample' => 'sample#index'
   get 'sample/group' => 'sample#group'
   get 'sample/monthly_report' => 'sample#monthly_report'
+
+  resources 'monthly_reports', except: :destroy
```
Comment (freezer530000) [inline config/routes.rb:9]: **[rails_best_practices]**
- restrict auto-generated routes monthly_reports (only: [:show, :index]) [:ref](http://rails-bestpractices.com/posts/2011/08/19/restrict-auto-generated-routes/)

**Comment by bakunyo** [comment]
これたしか古いやつだったと思うので、Closeしちゃいます〜

**Pull request closed without merging**